### PR TITLE
Release readiness [DEVINFRA-520] [DEVINFRA-521] [DEVINFRA-524] [DEVINFRA-525]

### PR DESCRIPTION
**Pull request by notoriaga**

### CODE REVIEW

**Review by john-michaelburke — APPROVED** [review]
LGTM!

File: piksi_tools/console/utils.py [hunk 1]
```diff
@@ -488,11 +488,9 @@
     IMU_CALIBRATION_ERROR: "Cal Error"
 }
 
-SMOOTHPOSE = 0
 DR_RUNNER = 1
 
 ins_type_dict = {
-    SMOOTHPOSE: "SP",
```
Comment (silverjam) [inline piksi_tools/console/utils.py:495]: We should keep this one, if they're using an old firmware, we'd still want to identify that smoothpose is active, see: https://swift-nav.atlassian.net/browse/DEVINFRA-526

**Review by silverjam — COMMENTED** [review]
Looks good, once tweak needed

File: piksi_tools/console/sbp_relay_view.py [hunk 2]
```diff
@@ -126,9 +124,10 @@ class SbpRelayView(HasTraits):
                               'network_refresh_button', show_label=False,
                               width=0.50),
                           show_border=True,
-                          label="Network"),
-                      )
-               )
+                          label="Network"
```
Comment (jayvdb) [inline piksi_tools/console/sbp_relay_view.py:127]: add trailing comma

File: piksi_tools/console/utils.py [hunk 3]
```diff
@@ -493,7 +493,7 @@
 
 ins_type_dict = {
     SMOOTHPOSE: "SP",
-    DR_RUNNER: "DR"
+    DR_RUNNER: "INS"
```
Comment (jayvdb) [inline piksi_tools/console/utils.py:496]: trailing comma

**Review by jayvdb — APPROVED** [review]

**Comment by notoriaga** [comment]
angular

Also adds the angular rate to the solutions table. Not sure if there is an accepted abbreviation for angular rate.

File: piksi_tools/console/settings.yaml [hunk 4]
```diff
@@ -1539,14 +1539,26 @@
   name: cog_output_min_speed
   expert: true
   type: float
-  digits: '1'
   units: Meters per second
+  digits: '2'
```
Comment (notoriaga) [inline piksi_tools/console/settings.yaml:1543]: @john-michaelburke turns out the `digits` here is actually what controls the number of display digits in the settings table so I added it back in :shrug:

Comment (john-michaelburke) [reply piksi_tools/console/settings.yaml:1543]: That's what I was thinking.

**Review by john-michaelburke — APPROVED** [review]
LGTM I think the other request was hyperlinks for the url in the update popup. Seemed like more of a nice to have though so nit. Let's ship it.

**Review by silverjam — APPROVED** [review]
:shipit:

**Pull request merged**